javascript/javascript2/week3/fun's-as-variables/funs-as-var.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Functions as vraiables</title>
+    <script>
+        debugger;
```

Comment:
we we need debugger inside script ? if it was for your local debugging then always remember debugger when you push your code